docs/reference/index.rst

```diff
@@ -14,6 +14,21 @@ apphook.
 
 If Aldryn Search, Haystack, et al, are not installed, this setting does nothing.
 
+The flag ``ALDRYN_NEWSBLOG_UPDATE_SEARCH_DATA_ON_SAVE``, when set to True
+(default value), updates the article's search_data field whenever the article
+is saved or a plugin is saved on the article's content placeholder. Set to false
+to disable this feature.
+
+
+*******************
+Management Commands
+*******************
+
+The management command: ``rebuild_article_search_data`` can be used to index all
```

Comment:
same, in the paragraph above you mentioned "updates the article's search_data field" this is perfect